sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_sessions_v2/metadata.yaml

```
@@ -4,7 +4,7 @@ description: |-
 owners:
 - kwindau@mozilla.com
 labels:
-  incremental: true
+  incremental: false
   owner1: kwindau@mozilla.com
 scheduling:
   dag_name: bqetl_google_analytics_derived_ga4
```

```suggestion
  dag_name: bqetl_google_analytics_derived_ga4
  date_partition_parameter: null
  parameters: ["submission_date:DATE:{{ds}}"]
```

also referenced tables need to be explictly listed for script using `referenced_tables`

---

Thank you, I did not know that!!